src/diffusers/models/prior_transformer.py

```diff
@@ -59,24 +59,50 @@ def __init__(
         num_embeddings=77,
         additional_embeddings=4,
         dropout: float = 0.0,
+        time_embed_act_fn: str = "silu",
+        norm_embedding_proj: bool = False,
+        time_embed_dim: Optional[int] = None,
+        clip_embedding_dim: Optional[int] = None,
```

Comment:
```suggestion
        embedding_proj_dim: Optional[int] = None,
```

Not all text encoders are clip, let's make it general